spec/index.bs

```diff
@@ -8041,6 +8110,16 @@ GPUSwapChain includes GPUObjectBase;
         1. Return |texture|.
 </div>
 
+<div algorithm>
+    To <dfn abstract-op lt='destroy the swap chain'>Destroy a swap chain</dfn>
+    for {{GPUSwapChain}} |swapChain| run the following steps:
+
+        1. If |swapChain|.{{GPUSwapChain/[[currentTexture]]}} is not `null` call
+            {{GPUTexture/destroy()}} on |swapChain|.{{GPUSwapChain/[[currentTexture]]}}.
+        1. Set |swapChain|.{{GPUSwapChain/[[currentTexture]]}} to `null`.
+        1. Make |swapChain| [=invalid=].
```

Comment:
I think the validity of the swap chain needs to be checked in getCurrentTexture. Probably inside "allocate a new swap chain texture", if the swap chain is invalid, the returned texture should be invalid and a validation error should be generated.